pkg/apis/rke.cattle.io/v1/certificaterotation.go

```diff
@@ -1,8 +1,6 @@
 package v1
 
 type RotateCertificates struct {
-	Generation int64 `json:"generation,omitempty"`
-
-	CACertificates bool     `json:"caCertificates,omitempty"`
-	Services       []string `json:"services,omitempty" norman:"type=enum,options=admin|api-server|controller-manager|scheduler|rke2-controller|rke2-server|cloud-controller|etcd|auth-proxy|kubelet|kube-proxy"`
+	Generation int64    `json:"generation,omitempty"`
+	Services   []string `json:"services,omitempty" norman:"type=enum,options=admin|api-server|controller-manager|scheduler|rke2-controller|rke2-server|cloud-controller|etcd|auth-proxy|kubelet|kube-proxy"`
```

Comment:
Do we still use norman types...?

Reply:
oh I didn't even think of that, it is harmless to add but ill remove it anyway